vaadin-crud-flow-demo/src/main/java/com/vaadin/flow/component/crud/CrudView.java

```diff
@@ -91,6 +92,26 @@ private void basicCrud() {
     }
     // end-source-example
 
+    private void setEditorPosition() {
+        // begin-source-example
+        // source-example-heading: Editor Position
+        Crud<Person> crud = new Crud<>(Person.class, createPersonEditor());
+
+        crud.setEditorPosition(CrudEditorPosition.BOTTOM);
```

Comment:
Because of the lack of space, this position is not looking good. It might make sense either:

- Change it to `ASIDE`
```diff		 
-		crud.setEditorPosition(CrudEditorPosition.BOTTOM);
+       crud.setEditorPosition(CrudEditorPosition.ASIDE);
```

- Or add some height to CRUD.
```diff
+		crud.setHeight("600px");
        crud.setEditorPosition(CrudEditorPosition.BOTTOM);
```

Reply:
Done.